src/Request/RequestFactoryInterface.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Psr\Http\Factory;
+namespace Psr\Http\Factory\Request;
```

Comment:
Do we really need all this name littering? All these namespaces with just one or two interfaces. I think keeping all factories in a single `Factory` sub-namespace was just fine.